### PR TITLE
feat: Implemented Popovics Constitutive Law

### PR DESCRIPTION
**Pull request by talledodiego**
Included Popovics-Mander  Constitutive Law and related test.

### CODE REVIEW

File: structuralcodes/materials/constitutive_laws.py [hunk 1]
```diff
@@ -461,6 +461,108 @@ def get_ultimate_strain(
         return (100, self._eps_cu1)
 
 
+class Popovics(ConstitutiveLaw):
+    """Class for Popovics-Mander constitutive law.
```
Comment (mortenengen) [inline structuralcodes/materials/constitutive_laws.py:465]: Please include a reference to the literature 👍

**Review by mortenengen — CHANGES_REQUESTED** [review]
Very good 😃 Please add references to the literature for the Popovics (and also the Sargin) model, and we are good.

File: structuralcodes/materials/constitutive_laws.py (hunk 1 above)
Comment (talledodiego) [reply structuralcodes/materials/constitutive_laws.py:465]: Done

**Comment by talledodiego** [comment]
Added both references. Thanks for the review!

**Review by mortenengen — APPROVED** [review]

**Pull request merged**